src/CodeCleaner/FunctionReturnInWriteContextPass.php

```diff
@@ -64,11 +55,8 @@ public function enterNode(Node $node)
                     continue;
                 }
 
-                $msg = ($node instanceof Isset_ && $this->atLeastPhp55) ? self::PHP55_MESSAGE : self::EXCEPTION_MESSAGE;
-                throw new FatalErrorException($msg, 0, E_ERROR, null, $node->getLine());
+                throw new FatalErrorException(self::ISSET_MESSAGE, 0, E_ERROR, null, $node->getLine());
```

Comment:
this still needs to distinguish between the two messages (based on `$node instanceof Isset_`).

we could prolly rename the constants to something clearer though :)

Reply:
Yeh, I messed this up, then didn't have time to fix it earlier today. I'll do it soon.

Reply:
Fixed, and updated the message to match the message PHP 7.0.0 and higher will spit out.